### PR TITLE
Add Highcharts Explorer

### PR DESCRIPTION
**Pull request by perkyguy**
This is adding an "explorer" to the highcharts translator. This allows you to change your inputs on a graph, and immediately see the result of the net.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,9 +3,16 @@ import RandomInput from './random-input';
 import Translate from './translate';
 import ValuesInput from './values-input';
 
-export {
+const brainCT = {
```
Comment (robertleeplummerjr) [inline src/index.js:6]: What if we put RandomInput, Translate, and ValuesInput as statics on BrainCT?

File: src/translate/highcharts/scanner.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
-export default function highcharts(results, options) {
+function informAboutDependencies(){
```
Comment (robertleeplummerjr) [inline src/translate/highcharts/scanner.js:1]: What do you think about putting each of these in their own directory, with documentation and image?

**Review by robertleeplummerjr — APPROVED** [review]
The recommendations I had were just for exploratory ideas

File: src/translate/highcharts/scanner.js (hunk 2 above)
Comment (perkyguy) [reply src/translate/highcharts/scanner.js:1]: documentation is definitely necessary, I'll throw together another PR for this pretty soon

File: src/index.js (hunk 1 above)
Comment (perkyguy) [reply src/index.js:6]: yeah, I messed up the `export` altogether, I'll throw in a smaller PR to fix this

**Pull request merged**